lib/orwell/web/router.ex

```diff
@@ -28,6 +28,8 @@ defmodule Orwell.Web.Router do
     delete "/auth", AuthController, :delete
     get "/auth/:provider", AuthController, :request
     get "/auth/:provider/callback", AuthController, :callback
+
+    get "/*path", PostController, :show
```

Comment:
First of all, yes, `/posts/something` seems like a no-brainer to me.

But why do you need the entire post path? Why not just, for example, the ID (`/posts/62`), which makes this look just like any regular REST route, and without the exploit possibilities you just mentioned?